### DISCUSSION TITLE
关于Client Id

### DISCUSSION THREAD

**Discussion by Miaoyww** [upbit/pixivpy#205] [Q&A] [answered]
我是一个Python的初学者，浏览过作者的代码之后，我心里有一个疑惑
那就是关于 client_id,client_secret与hash_secret 的获取方法
我在搜索引擎上寻找了一番，无果之后只好来这里来问一下了
希望作者您能抽时间回复一下，十分感谢

**Comment by Mikubill** [comment, marked as answer]
client_id, client_secret与hash_secret都是解包Android APK得到的。具体获得方法可以参考下面的链接

ref: https://github.com/upbit/pixivpy/issues/83#issuecomment-527163914 https://github.com/upbit/pixivpy/issues/92#issuecomment-546011823  dup: https://github.com/upbit/pixivpy/issues/98  https://github.com/upbit/pixivpy/issues/121

**Reply by Miaoyww** [reply]
好的！十分感谢🙏
